### ISSUE TITLE
Erratas pushed to stable twice with different metadata

### ISSUE THREAD

**Issue by wiad** [fedora-infra/bodhi#4521]
For a week or so I have noticed that erratas that are pushed to EPEL stable seem to have old metadata from the 'testing' phase, for example the attribute `status` is set to `testing` and `issued_date` is set to the date when the patch went to 'testing', not the date of pushing it to stable. A day or two later a new errata is published where metadata is updated - `"status": "stable"` and `issued_date` is updated. 

Example:
FEDORA-EPEL-2022-f64d777807 was published in stable 2022-05-20, with `status: testing` and `issued_date: 2022-05-12 03:16:55`. The same errata was then published again 2022-05-23 with `status: stable` and `issued_date: 2022-05-20 02:26:36`. 
So it seems that the errata metadata is not updated before it is being pushed to stable. 

We sync the EPEL repo with Pulp and it is via Pulp I have inspected the errata metadata. We only sync the stable repo, not the testing repo. This causes some problems for us since we trigger actions when important errata arrives, and now those actions are triggerd twice for the same errata.

**Comment by wiad** [comment]
Any feedback on this? 
This is still happening.

**Comment by wiad** [comment]
This is still happening, for example with FEDORA-EPEL-2022-89ad385971. 
We got the errata 2022-09-21, with metadata:
``` 
"issued_date": "2022-07-20 01:45:57",
"status": "testing"
```

then got an updated errata 2022-09-22, with metadata:
```
"issued_date": "2022-09-20 00:36:48",
"status": "stable"
```

**Comment by mattiaverga** [comment]
I thin this was fixed in Bodhi 7.0.0 but I forgot to link this bug. I'm going to close this, feel free to reopen if the problem persists.
